src/Features/Core/Portable/EditAndContinue/EditAndContinueDiagnosticDescriptors.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    internal static class EditAndContinueDiagnosticDescriptors
+    {
+        private const int GeneralDiagnosticBaseId = 1000;
+        private const int ModuleDiagnosticBaseId = 2000;
+        private static readonly int s_diagnosticBaseIndex;
+
+        private static readonly LocalizableResourceString s_rudeEditLocString;
```

Comment:
💡 The other `LocString` suffix symbols in this file would benefit from a similar change

Reply:
The first two are used in titles the third is used as message. The fields are named after the corresponding resource strings.

---
In reply to: [270628184](https://github.com/dotnet/roslyn/pull/34027#discussion_r270628184) [](ancestors = 270628184)